testing/test_package/lib/example.dart

```diff
@@ -162,6 +162,7 @@ class ConstantCat implements Cat {
 }
 
 /// implements [Cat], [E]
+/// {@example core/pipes/ts/slice_pipe/slice_pipe_example.ts region='SlicePipe_list'}
```

Comment:
What is this path relative to? We might want to document that fact in the main source for this feature?